diagrams.md

```
@@ -0,0 +1,907 @@
+skinparam object {
+    BackgroundColor #F0F8FF
+    ArrowColor #2C3E50
+    BorderColor #2C3E50
+}
+
+' Database Objects
+package "Database" {
+    object "user: User" as user {
+        id = 1
+        name = "Remon Buddhacharya"
+        email = "user@example.com"
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)